lib/main.dart

```diff
@@ -144,7 +144,11 @@ class _TenTenOneState extends State<TenTenOneApp> {
 
   Future<void> setupRustLogging() async {
     api.initLogging().listen((event) {
-      FLog.logThis(text: 'log from rust: ${event.msg}', type: LogLevel.DEBUG);
+      // Only log to Dart file in release mode - in debug mode it's easier to
+      // use stdout
+      if (foundation.kReleaseMode) {
+      FLog.logThis(text: '${event.target}: ${event.msg}', type: LogLevel.DEBUG);
```

Comment:
❓ have you tried setting the log level off an on again for the rust logs? that way the logs would still end up in the database and are not printed to stdout. I think the level would still be helpful when reviewing the logs.

Reply:
feel free to submit a follow-up; I did only the amount I needed to get me unblocked :)